grafana/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM grafana/grafana:6.1.3
+
+ADD ./provisioning /etc/grafana/provisioning
```

Comment:
We will use [Grafana Provisioning](https://grafana.com/docs/administration/provisioning/) to provide our dashboards automatically.

Grafana reads yaml files located at `/etc/grafana/provisioning` to provide dashboards, so we gotta add those files to our grafana image.